store/cachekv/store.go

```diff
@@ -93,6 +93,32 @@ func (store *Store) Delete(key []byte) {
 	store.setCacheValue(key, nil, true)
 }
 
+type cEntry struct {
+	key string
+	val *cValue
+}
```

Comment:
This type is only used inside the `Write` method -- declaring the type inside the method will improve readability, by keeping the declaration as close as possible to its only use, and by reducing the number of package-level declarations.